apps/emqx_management/src/emqx_mgmt_cluster.erl

```diff
@@ -0,0 +1,199 @@
+%%--------------------------------------------------------------------
+%% Copyright (c) 2024 EMQ Technologies Co., Ltd. All Rights Reserved.
+%%
+%% Licensed under the Apache License, Version 2.0 (the "License");
+%% you may not use this file except in compliance with the License.
+%% You may obtain a copy of the License at
+%%
+%%     http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing, software
+%% distributed under the License is distributed on an "AS IS" BASIS,
+%% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+%% See the License for the specific language governing permissions and
+%% limitations under the License.
+%%--------------------------------------------------------------------
+
+-module(emqx_mgmt_cluster).
+
+-behaviour(gen_server).
+
+%% APIs
+-export([start_link/0]).
+
+-export([invite_async/1, invitation_view/0]).
+
+%% gen_server callbacks
+-export([
+    init/1,
+    handle_call/3,
+    handle_cast/2,
+    handle_info/2,
+    terminate/2,
+    code_change/3
+]).
+
+%%--------------------------------------------------------------------
+%% APIs
+%%--------------------------------------------------------------------
+
+start_link() ->
+    gen_server:start_link({local, ?MODULE}, ?MODULE, [], []).
+
+-spec invite_async(atom()) -> ok | ignore | {error, {already_started, pid()}}.
+invite_async(Node) ->
+    JoinTo = node(),
+    case Node =/= JoinTo of
+        true ->
+            gen_server:call(?MODULE, {invite_async, Node, JoinTo});
+        false ->
+            ignore
```

Comment:
Nit: mb. `{error, same_node}` for explicitness?

Reply:
I think we can keep this kind of implementation for now. The main reason is that other functions in this module also return ignore, and the underlying ekka:join also returns ignore to indicate that it cannot join itself.